charts/camunda-platform/values.yaml

```diff
@@ -2633,7 +2635,7 @@ connectors:
     ## @param connectors.image.repository defines which image repository to use
     repository: camunda/connectors-bundle
     ## @param connectors.image.tag can be set to overwrite the global tag, which should be used in that chart
-    tag: 8.4.3
+    tag: 8.4.4
```

Comment:
@aabouzaid Is this allowed? Would this not effect our release process?

Reply:
@hamza-m-masood Yes, due to the several patches we had to roll out last time, the Connectors versions moved ahead of the platform patch version. 

For example: For Camunda 8.3.6 the Connectors are on 8.3.7. We shop that we will align with the upcoming patches here.

Reply:
Hi @sbuettner we also have an open renovate [PR](https://github.com/camunda/camunda-platform-helm/pull/1301) with the same changes.

Reply:
I'd propose to close the renovate PR as this version bump requires the changes from this PR